quic/s2n-quic-core/src/event.rs

```diff
@@ -113,6 +120,12 @@ impl IntoEvent<Timestamp> for crate::time::Timestamp {
     }
 }
 
+impl From<crate::time::Timestamp> for Timestamp {
```

Comment:
I added this just to be able to use the mock_clock implementation for `time::Timestamp`. Is there a better way? Should it be behind a `cfg(test)` macro?